2996_reflection/reflection.md

```diff
@@ -3541,6 +3541,8 @@ constexpr int d = template [:^^TCls:]<int>::b;  // template [:^^TCls:]<int> is n
 
 template <auto V> constexpr int e = [:V:];   // splice-expression
 constexpr int f = template [:^^e:]<^^S::a>;  // splice-expression
+
+auto g = typename [:^^int:](42);  // [:^^int:] is not an expression
```

Comment:
Is this... supposed to be an error? I don't know what this comment means.

Reply:
Not an error - Let me know if this is clearer.